gatsby-config.js

```diff
@@ -100,20 +100,20 @@ module.exports = {
           },
         ],
       },
-      //   {
-      //     title: "Concepts",
-      //     path: "/guides/concepts/customization.md",
-      //     pages: [
-      //       {
-      //         title: "UI Customization",
-      //         path: "/guides/concepts/customization.md",
-      //       },
+       {
+      title: "Concepts",
+      path: "/guides/concepts/migration.md",
+      pages: [
+      {
+      title: "Migration Guide: Adobe Express Embed SDK V3 to V4",
```

Comment:
I was wondering whether to use a shorter title (just in the navbar) that fits a single row, like "Migration Guide: V3 to V4," leaving the full title on the page.

<img width="212" alt="image" src="https://github.com/user-attachments/assets/bf22e14a-7940-4fb3-a556-d9269cb29e21">